### ISSUE TITLE
New Equinox-based Vision Repository

### ISSUE THREAD

**Issue by bhyun-kim** [patrick-kidger/equinox#639]
Hi Patrick, 

I am building a repository of image classification models with Equinox as a personal project. I have added ResNet and LeNet so far and plan to add more. 

While EqxVision doesn't have training script, I will provide training script with all the hyperparameters. I hope this repo to contribute to this ecosystem. 

https://github.com/bhyun-kim/qvix/tree/main

**Comment by patrick-kidger** [comment]
That's awesome!

In fact, you've inspired me to add a new show-and-tell label, just for this. :)

I look forward to seeing how this project grows.
